app/controllers/idv/review_controller.rb

```diff
@@ -50,8 +50,11 @@ def new
 
     def create
       irs_attempts_api_tracker.idv_password_entered(success: true)
-      log_reproof_event if Profile.find_by(id: idv_session&.profile_id)&.active?
+
       init_profile
+
+      log_reproof_event if idv_session.profile.has_proofed_before?
```

Comment:
This was the bug: The user has no active `idv_session.profile_id` until we call `init_profile`, so `log_reproof_event` would never get called. I had moved this check above `init_profile` while working in the test and didn't come back to run through it manually. 🤦‍♂️